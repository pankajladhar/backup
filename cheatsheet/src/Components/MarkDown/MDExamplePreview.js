import React from 'react';
import PropTypes from 'prop-types';

const MDExamplePreview = (props) => {
    return (
        <div className="MDExample__Preview markdown-body">
            <div className="MDExample__Title">Preview</div>
            <div className="MDExample__Content">
                {props.children}
            </div>
        </div>
    )
}

MDExamplePreview.propTypes = {
    children: PropTypes.node
}


export default MDExamplePreview;
