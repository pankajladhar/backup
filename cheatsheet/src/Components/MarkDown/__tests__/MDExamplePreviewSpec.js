import React from 'react';
import renderer from 'react-test-renderer';
import MDExamplePreview from './../MDExamplePreview';

describe('<MDExamplePreview />', () => {
    it('should render properly', () =>{
        let tree = renderer.create(
            <MDExamplePreview>
                <span>MDExamplePreview</span>
            </MDExamplePreview>
        ).toJSON();

        expect(tree).toMatchSnapshot()
    })
})