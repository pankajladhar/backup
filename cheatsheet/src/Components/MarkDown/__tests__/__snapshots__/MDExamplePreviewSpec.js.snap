// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MDExamplePreview /> should render properly 1`] = `
<div
  className="MDExample__Preview markdown-body"
>
  <div
    className="MDExample__Title"
  >
    Preview
  </div>
  <div
    className="MDExample__Content"
  >
    <span>
      MDExamplePreview
    </span>
  </div>
</div>
`;
